packages/cubejs-databricks-jdbc-driver/src/DatabricksDriver.ts

```diff
@@ -703,6 +703,13 @@ export class DatabricksDriver extends JDBCDriver {
     // wasbs://real-container-name@account.blob.core.windows.net
     // The extractors in BaseDriver expect just clean bucket name
     const url = new URL(this.config.exportBucket || '');
+    const prefix = url.pathname.slice(1);
+    let delimiter = '';
+    if (prefix && !prefix.endsWith('/')) {
```

Comment:
I prefer to write such ifs using a ternary operator because it allows me to mark variables with a `const` modifier.

Reply:
Funny, that I fix it to `obj?.smth` everywhere I faced it :) But missed in my own code)